### PR TITLE
User-delete-entry - Heerak Moon

### PR DESCRIPTION
**Pull request by heerakmoon**
https://user-images.githubusercontent.com/113075380/220604945-372dbab1-a2ac-42c7-891c-b686d89a3e17.mp4

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -147,5 +152,40 @@ $ul.addEventListener('click', function (event) {
...
+  }
+  var removeLi = document.getElementById(liIdNum);
+  removeLi.remove();
+  $form.reset();
+  $h1.textContent = 'New Entry';
+  $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  toggleNoEntries();
+  $modal.className = 'hidden';
+  viewSwap('entries');
+  data.editing = null;
```
Comment (robgmerrill) [inline js/main.js:190]: One last thing. I didn't see your local storage in the submission video but I'm guessing that `data.nextEntryId` isn't being lowered by one every time an item is being deleted. To do that you would just decrement the value by one when the confirm button is clicked.

`data.nextEntryId--`

**Review by robgmerrill — CHANGES_REQUESTED** [review]
One last request! Because I didn't see local storage in the submission video I couldn't see if `data.nextEntryId` is being reduced by 1 for each deleted item. If it is, please send me a gif. If not, I commented in the file where I think you can make that update. Slack me any questions if that isn't clear!

**Comment by heerakmoon** [comment]
https://user-images.githubusercontent.com/113075380/220840202-bb84932b-51e6-4ba6-9340-bb31af81bf70.mp4

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**